### PR TITLE
#82 | Apply Upgrades to Player

### LINKED ISSUE #82: Integrate Upgrades into global create player

### PR DESCRIPTION
**Pull request by demwilson**
* Starting player stats and items based on upgrades object.
* Fixed typo.
* Added a bunch of new items.

### CODE REVIEW

File: Global.gd [hunk 1]
```diff
@@ -98,11 +102,21 @@ func _ready():
 		Move.new("Heal", 1, Move.MoveType.HEAL, Move.AnimationPath.FIREBOLT, 1, 2, [0, 2]),
 	]
 	var available_items = [
-		Item.new(0, "Basic Attack", Item.ItemTier.LEVEL_ONE, Item.ItemType.MOVE, "This is a basic attack.", Moves.MoveList.BASIC_ATTACK),
+		Item.new(0, "Crowbar", Item.ItemTier.GAME_START, Item.ItemType.MOVE, "You swing the crowbar.", Moves.MoveList.BASIC_ATTACK),
 		Item.new(1, "Flimsy Sword", Item.ItemTier.LEVEL_ONE, Item.ItemType.BONUS, "This is an almost useless sword.", [Creature.Stats.ATTACK, 1]),
-		Item.new(2, "Cybernetic Eye", Item.ItemTier.LEVEL_ONE, Item.ItemType.STAT, "This eye knows where things are even if you don't.", [Creature.Stats.ACCURACY, 2]),
-		Item.new(3, "Firebolt", Item.ItemTier.LEVEL_ONE, Item.ItemType.MOVE, "This launches a bolt of fire at your enemy!", Moves.MoveList.FIREBOLT),
-		Item.new(4, "Friendly Robot Servant", Item.ItemTier.LEVEL_ONE,Item.ItemType.ALLY, "This robot will fight for you.", Enemies.EnemyList.ROBOT_T1),
+		Item.new(2, "Ruler", Item.ItemTier.LEVEL_ONE, Item.ItemType.BONUS, "Helps with measurements.", [Creature.Stats.ACCURACY, 1]),
```
Comment (beserkzombie) [inline Global.gd:107]: I see some magic numbers!

Comment (demwilson) [reply Global.gd:107]: OH NOES!

Comment (beserkzombie) [reply Global.gd:107]: REJECTED!

Comment (beserkzombie) [reply Global.gd:107]: Will fix later when you are not being so obstinate

Comment (demwilson) [reply Global.gd:107]: 👍

**Review by beserkzombie — APPROVED** [review]
Developer extra obstinate. just ship it, screw it.

**Pull request merged**